src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -74,26 +77,30 @@ export default class GroupComparisonMutationsTabPlot extends React.Component<
     }
 
     @autobind
-    protected countUniqueMutations(mutations: Mutation[]) {
+    protected countUniqueMutations(mutations: Mutation[], group: string) {
         return this.axisMode === AxisScale.PERCENT
             ? (countUniqueMutations(mutations) /
-                  this.props.store.profiledSamplesCount.result!) *
+                  this.props.store.groupToProfiledSamples.result![group]
+                      .length) *
                   100
             : countUniqueMutations(mutations);
     }
 
     @action.bound
     private onScaleToggle(selectedScale: AxisScale) {
+        this.props.urlWrapper.updateURL({
```

Comment:
if we we are updaing the url then we shouldn't need to also need the local (this.axisMode).  That hsould instead be derived from the url